sections/strategy.tex

```diff
@@ -0,0 +1,19 @@
+\chapter{Strategy}\label{Metode}
+
+This chapter will briefly explain the strategy we will use to find an answer to our problem statement. 
+Developing a compiler is a complex and time consuming process, and having a strategy for the development is a huge help, which is why this is discussed in the paper.
+
+\section{Scrum}
+
+Developing a compiler is as mentioned complex and time consuming, but in our case especially it also has a great deal of uncertainty. 
+Not only because we need to design the programming language, and create the rules of the language, but also because we are very new to the world of compiler development.
+This means that we will need to follow to courses on this semester for a while, learning the skills needed to develop a compiler as we do not know these yet.
+When a project has a lot of uncertainty, like ours has, it is often a smart choice to have an agile development method, compared to having a linear development method such as the waterfall model.
+Therefore we choose to develop this project according to Scrum, but with some changes.
+We will make use of scrum's organisational tool, such as the scrumboard, daily scrums and sprints with stories.
+We will not have the specific roles which you would normally find in scrum, such as the scrum master or the product owner. \citep{Scrum}
+The reason for this is, partly due to development methods not being in the curriculum for this project, but also due to the project not having use of these tools.
+There is no customer for whom we are developing the compiler nor is there really time for one person in the group to be scrum master, as he would still need to focus a lot on the tasks the rest of the group are also partaking in.
+
+With this method we are able to work in short sprints of one to two weeks, learning and using what we learn in the following sprints, instead of planning out the entire project from the start with minimum knowledge of the project.
```

Comment:
Lidt uklar: "learning and using what we learn in the following sprints" Måske noget ala: "which gives us the opputunity to make incorporate the tools, concepts and methods we learn along the ways in our development of the language and compuler"